### PR TITLE
Fixed #27463 Fix ambiguous variable flake8 warnings

### PR DESCRIPTION
**Pull request by RaminFP**

### CODE REVIEW

File: django/contrib/admin/options.py [hunk 1]
```diff
@@ -339,11 +339,11 @@ def lookup_allowed(self, lookup, value):
         # Check FKey lookups that are allowed, so that popups produced by
         # ForeignKeyRawIdWidget, on the basis of ForeignKey.limit_choices_to,
         # are allowed to work.
-        for l in model._meta.related_fkey_lookups:
+        for L in model._meta.related_fkey_lookups:
```
Comment (timgraham) [inline django/contrib/admin/options.py]: A better improvement throughout the PR would be to move away from one letter variable names, e.g. `lookup` here.

Comment (RaminFP) [inline django/contrib/admin/options.py]: Hi @timgraham, i think you use https://pypi.python.org/pypi/flake8 latest update `2016-08-08`,
Yes, there is a warnings `E305 expected 2 blank lines after class or function definition, found 1` in many files,you can allow to me for fix it?
https://code.djangoproject.com/ticket/27463#comment:10

Comment (timgraham) [inline django/contrib/admin/options.py]: Yes, I guess we'll want to fix those errors.

**Comment by RaminFP** [comment]
@timgraham ready for you!

**Comment by RaminFP** [comment]
@timgraham please review,and merge for avoid conflict files, Thanks,

**Comment by timgraham** [comment]
Partly merged in 967be82443b5640d61608a89897d8ce2bc44fa54. I'm stlil not sure how to get flake8 to issue the ambiguous variable warnings... have it not incorporated that warning in the latest release?

**Comment by RaminFP** [comment]
Now release flake8 https://github.com/PyCQA/flake8/releases i use pystylecode latest version for detect new warnings,

**Comment by RaminFP** [comment]
@timgraham what the problems in fix?

**Comment by RaminFP** [comment]
@timgraham last comment and please answer https://code.djangoproject.com/ticket/27483,

**Comment by RaminFP** [comment]
@timgraham still flake8 use version pycodestyle 2.0.0 no pycodestyle 2.1.0 (pyflakes 1.2.3) also pyflakes 1.2.3 (https://github.com/PyCQA/pyflakes/releases),
please merge i believe there isn't problem,

**Comment by timgraham** [comment]
The warnings aren't triggered due to https://github.com/PyCQA/flake8/blob/64cbc6559d7b5ee27ce9206e19ef038c05f423b4/setup.py#L101-L102. Anyway, I fixed this up and will merge when the build passes.

Please don't use GitHub to ping me about unrelated issues. Thanks.

**Comment by timgraham** [comment]
buildbot, test this please.

**Comment by timgraham** [comment]
buildbot, test this please.

**Pull request merged**